_capabilities/fault_lines_UCLA

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
Can you make this a Markdown file (i.e. fault_lines_UCLA.md)? This will allow it to render correctly on the website.

Reply:
done